src/main/drivers/bus_spi_ll.c

```diff
@@ -518,17 +548,17 @@ void spiSequence(const extDevice_t *dev, busSegment_t *segments)
             dmaSafe = false;
             break;
         }
-#elif defined(STM32F4)
+#elif defined(STM32G4)
         // Check if RX data can be DMAed
         if ((checkSegment->rxData) &&
-            // DTCM can't be accessed by DMA1/2 on the F4
-            (IS_DTCM(checkSegment->rxData)) {
+            // CCM can't be accessed by DMA1/2 on the G4
+            IS_CCM(checkSegment->rxData)) {
             dmaSafe = false;
             break;
         }
         if ((checkSegment->txData) &&
-            // DTCM can't be accessed by DMA1/2 on the F4
-            (IS_DTCM(checkSegment->txData)) {
+            // CCM can't be accessed by DMA1/2 on the G4
+            IS_CCM(checkSegment->txData)) {
```

Comment:
```suggestion
            IS_CCM(checkSegment->txData) {
```

Reply:
@haslinghuis I don't think this is the right change, it's in `&&`, see lines above.

Reply:
Thanks you are right. Updated the suggestion !!!

Reply:
It could do without the inner parenthesis though - but then we have to change line 559 too.

Reply:
As per the above, the old code, quoted below, was wrong, but never compiled.

```
#elif defined(STM32F4)
        // Check if RX data can be DMAed
        if ((checkSegment->rxData) &&
            // DTCM can't be accessed by DMA1/2 on the F4
            (IS_DTCM(checkSegment->rxData)) {
            dmaSafe = false;
            break;
        }
```